tests/tape/interfaces/test_qnode_autograd.py

```diff
@@ -440,7 +440,32 @@ def circuit(a, p):
 
     def test_probability_differentiation(self, dev_name, diff_method, tol):
         """Tests correct output shape and evaluation for a tape
-        with prob and expval outputs"""
+        with a single prob output"""
+
+        dev = qml.device(dev_name, wires=2)
+        x = np.array(0.543, requires_grad=True)
+        y = np.array(-0.654, requires_grad=True)
+
+        @qnode(dev, diff_method=diff_method, interface="autograd")
+        def circuit(x, y):
+            qml.RX(x, wires=[0])
+            qml.RY(y, wires=[1])
+            qml.CNOT(wires=[0, 1])
+            return qml.probs(wires=[1])
+
+        res = qml.jacobian(circuit)(x, y)
+
+        expected = np.array(
+            [
+                [-np.sin(x) * np.cos(y) / 2, -np.cos(x) * np.sin(y) / 2],
+                [np.cos(y) * np.sin(x) / 2, np.cos(x) * np.sin(y) / 2],
+            ]
+        )
+        assert np.allclose(res, expected, atol=tol, rtol=0)

```

Comment:
I like to use `np.testing.assert_allclose`. When the test fails, it gives you a cleaner output as to what went wrong.

Reply:
```suggestion
        assert np.testing.assert_allclose(res, expected, atol=tol, rtol=0)
```

Reply:
Oops, turns out this can't be changed here since `qml.np.testing` doesn't exist. A good reason to import both numpy's separately in the tests.